src/content/components/uikit/Breadcrumbs/index.ts

```diff
@@ -0,0 +1,70 @@
+import dynamic from 'next/dynamic';
+
+import {Repos} from '../../../../types/common';
+import {getGithubUrl, getReadmeUrl} from '../../utils';
+
+const getterOptions = {repoName: Repos.Uikit, componentName: 'Breadcrumbs'};
+
+export const breadcrumbsConfig = {
+    id: 'breadcrumbs',
+    title: 'Breadcrumbs',
+    githubUrl: getGithubUrl(getterOptions),
+    content: {
+        readmeUrl: getReadmeUrl(getterOptions),
+    },
+    sandbox: {
+        component: dynamic(() =>
+            import('./BreadcrumbsWrapper').then((mod) => mod.BreadcrumbsWrapper),
+        ),
+        props: {
+            items: {
+                type: 'object',
```

Comment:
It's unsupported `type`, can you move this prop definition to `BreadcrumbsWrapper`?

Reply:
Fixed, thx